Multisig/UI/Safe Management/Create Safe/Setup Safe/CreateSafeFormUIModel.swift

```diff
@@ -119,20 +119,24 @@ class CreateSafeFormUIModel {
     }
 
     var isCreateEnabled: Bool {
-        (state == .ready || (state == .keyNotFound && userSelectedPaymentMethod == .relayer)) &&
+        (state == .ready || (state == .keyNotFound && userSelectedRelayerAndItIsAvailableForChain)) &&
         name != nil &&
         !name!.isEmpty &&
         chain != nil &&
         !owners.isEmpty &&
         threshold > 0 &&
         threshold <= owners.count &&
-        (selectedKey != nil || userSelectedPaymentMethod == .relayer) &&
+        (selectedKey != nil || userSelectedRelayerAndItIsAvailableForChain) &&
         transaction != nil &&
-        (deployerBalance != nil || userSelectedPaymentMethod == .relayer) &&
-        (deployerBalance! >= transaction.requiredBalance || userSelectedPaymentMethod == .relayer) &&
+        (deployerBalance != nil || userSelectedRelayerAndItIsAvailableForChain) &&
+        (deployerBalance! >= transaction.requiredBalance || userSelectedRelayerAndItIsAvailableForChain) &&
         error == nil
     }
 
+    var userSelectedRelayerAndItIsAvailableForChain: Bool {
+        userSelectedPaymentMethod == .relayer && chain.isSupported(feature: .relayingMobile)
```

Comment:
userSelectedPaymentMethod should be either .relayer or .signer. Relayer option is assignable only if chain supports it.

Reply:
+1

Reply:
That is not how it is ATM. You mean that is how it should be? Maybe we need a third value: `.notSelected` and that should be the initial value?